src/app/overview/transactions-view/transactions-view.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { ExplorerModel } from '../../explorer/explorer.model';
+import { ExplorerService } from '../../explorer/explorer.service';
+import { WalletModel } from '../../wallet/wallet.model';
+import { WalletService } from '../../wallet/wallet.service';
+import { RpcSend } from '../../rpc/rpc-send.model';
+import { RpcReceive } from '../../rpc/rpc-receive.model';
+
+export interface SendToAddressModel {
+  amount: Number;
+  destinationAddress: String;
+  feeIncluded: Boolean;
+}
+
+
+@Component({
+  selector: 'app-transactions-view',
+  templateUrl: './transactions-view.component.html',
+  styleUrls: ['../overview.component.css','./transactions-view.component.css']
+})
+export class TransactionsViewComponent implements OnInit {
+
+  explorer: ExplorerModel;
```

Comment:
We can remove `explorer`, `wallet`, `ExplorerService` `transaction` `qrMainAddress`